src/lib/sedlexing.ml

```diff
@@ -133,9 +128,7 @@ let refill lexbuf =
   let n = lexbuf.refill lexbuf.buf lexbuf.pos chunk_size in
   if n = 0 then lexbuf.finished <- true else lexbuf.len <- lexbuf.len + n
 
-let new_line lexbuf =
-  if lexbuf.curr_line != 0 then lexbuf.curr_line <- lexbuf.curr_line + 1;
-  lexbuf.curr_bol <- lexbuf.pos + lexbuf.offset
+let new_line lexbuf = lexbuf.curr_line <- lexbuf.curr_line + 1
```

Comment:
It has been years since I looked/used sedlex, so perhaps I'm missing something obvious, but why did you drop `lexbuf.curr_bol <- lexbuf.pos + lexbuf.offset`?

Reply:
My bad, adding it back.